pkg/cmd/start/bootstrap.go

```diff
@@ -85,7 +86,10 @@ func bootstrap(ctx context.Context) manager.Manager {
 	serveCRMetrics(ctx, cfg, namespace)
 	createMetricsService(ctx, cfg, namespace)
 	detectOAuthProxyImageStream(ctx, mgr)
-
+	err = opmetrics.Bootstrap(ctx, namespace, mgr.GetClient())
+	if err != nil {
+		log.WithError(err).Error("Failed to initialize metrics")
```

Comment:
Errors in Go shouldn't start with a capital letter, except if they are the highest level (ie, at the main.go for the program).